scalableminds/webknossos
# Don't re-render voulme tool bar unnecessarily

This PR changes the volume tool bar to only depend on the active tool to avoid unnecessary re-rendering.
### URL of deployed dev instance (used for testing):
- https://avoidunnecessaryrerender.webknossos.xyz

### Steps to test:
- Open a volume tracing and turn on `highlight updates` in the React Dev Tools.
- Use the `Trace` and `Brush` Tool. The usage should not trigger a re-render of the volume tool bar (buttons: `Move`, `Trace`, `Brush` and `New Cell`).

### Issues:
- fixes #3911

------
